client/app/core/rbac.service.js

```diff
@@ -21,4 +32,91 @@ export function RBACFactory() {
   function all() {
     return features;
   }
+
+  function setNavigation(productFeatures) {
+    const features = {
+      dashboard: {show: entitledForDashboard(productFeatures)},
+      services: {show: entitledForServices(productFeatures)},
+      orders: {show: entitledForServices(productFeatures)},
+      requests: {show: entitledForRequests(productFeatures)},
+      catalogs: {show: entitledForServiceCatalogs(productFeatures)},
+      dialogs: {show: entitledForService(productFeatures)},
+      administration: {show: entitledForService(productFeatures)},
+    };
+
+    return features;
```

Comment:
Still returning on a setter, unless this isn't a setter, in which case it should be renamed.